File: backend/src/main/java/org/icpclive/events/WF/json/WFProblemInfo.java
package org.icpclive.events.WF.json;

import org.icpclive.events.ProblemInfo;

/**
 * Created by Aksenov239 on 3/5/2018.
 */
public class WFProblemInfo extends ProblemInfo {
    public int id;
    public int testCount;
    public int[] submissions;

    public WFProblemInfo(int languages) {
        submissions = new int[languages];
    }
}

File: backend/src/main/kotlin/org/icpclive/events/WF/json/WFProblemInfo.kt
package org.icpclive.events.WF.json

import org.icpclive.events.ProblemInfo

/**
 * Created by Aksenov239 on 3/5/2018.
 */
class WFProblemInfo(languages: Int) : ProblemInfo() {
    @JvmField
    var id = 0
    @JvmField
    var testCount = 0
    @JvmField
    var submissions: IntArray

    init {
        submissions = IntArray(languages)
    }
}